.github/workflows/central-publish.yml

```
@@ -13,8 +13,31 @@ on:
           - STAGE CENTRAL
 
 jobs:
+  run_tests:
+    name: Run Tests
+    runs-on: ubuntu-22.04
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)